sys/include/fs/fatfs.h

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (C) 2017 HAW-Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_fatfs  FatFs integration
+ * @ingroup     sys_fs
+ * @{
+ *
+ * @file
+ * @brief       FatFs integration for vfs
+ *
+ * @author      Michel Rottleuthner <michel.rottleuthner@haw-hamburg.de>
+ */
+
+#ifndef FATFS_H
+#define FATFS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef VFS_DIR_BUFFER_SIZE
+#define VFS_DIR_BUFFER_SIZE (44)
```

Comment:
This might not work as expected

Reply:
thanks for pointing this out. Its fixed now.